src/pages/Login.vue

```diff
@@ -395,16 +397,32 @@ export default {
       // whether the phone number entered by the user is valid
       return this.phoneInput.toString().match(/^([0]|\+91)?[6-9]\d{9}$/g) != null;
     },
+    startCountdown(seconds) {
+      // counts seconds before enabling resend OTP button
+      this.counter = seconds;
+      const interval = setInterval(() => {
+          this.counter--;
+          if(this.counter == 0)

```

Comment:
plus, have you set up your Vue linter and `pre-commit`? - issues of wrong spacing (between `if` and `(this.counter == 0)`) should be fixed before the commit is made itself so that we don't have to point it out.

Reply:
Yes I checked for pre-commit

Reply:
No, I don't have vue linter.